CHANGES_NEXT_RELEASE

```diff
@@ -8,4 +8,5 @@
 - Fix: sanity check for string query 'q' - detect 'left-hand-side missing' (Issue #1754)
 - Fix: more sanity checks for string query 'q' (q empty, parts of 'q' empty - parts of 'q' are separated by ';')
 - Fix: error message when updating attribute value for two entities with same id (Issue #1387)
- 
+- Add: sanity checks for numbers (Issue #1306)
```

Comment:
Maybe this should be "Hardening:" ? 

("Add:" is typically used for new features)